.github/workflows/ci.yml

```
@@ -34,3 +34,18 @@ jobs:
 
       - name: Check formatting
         run: crystal tool format --check
+
+  testfreebsd:
+    runs-on: macos-latest
+    name: Test (freebsd)
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install Crystal and test
+      id: test
+      uses: vmactions/freebsd-vm@v0.1.3
+      with:
+        usesh: true
+        prepare: pkg install -y libyaml libevent crystal shards
+        run: |
+          shards install
+          make test
```

```suggestion
        prepare: pkg install -y libyaml libevent crystal shards gmake
        run: |
          shards install
          gmake test
```

Even if the current makefile works for both, GNU and BSD dialects, it's better to just use GNU make everywhere.
See https://github.com/crystal-ameba/ameba/pull/221#issuecomment-812908622